rest/route/host.go

```diff
@@ -413,6 +413,11 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 			"context": "user offboarding",
 			"user":    ch.user,
 		}))
+		grip.Error(message.WrapError(user.ClearUserSettings(ch.user), message.Fields{
```

Comment:
Should this be added to the catcher rather than just logged?

Reply:
done